frontend/components/dashboard/table/SummaryValidators.vue

```diff
@@ -0,0 +1,155 @@
+<script setup lang="ts">
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import {
+  faArrowUpRightFromSquare,
+  faPowerOff
+} from '@fortawesome/pro-solid-svg-icons'
+import type { DashboardValidatorContext, SummaryTimeFrame } from '~/types/dashboard/summary'
+import { DashboardValidatorSubsetModal } from '#components'
+import { getGroupLabel } from '~/utils/dashboard/group'
+import type { DashboardKey } from '~/types/dashboard'
+import type { VDBSummaryValidators } from '~/types/api/validator_dashboard'
+
+interface Props {
+  // TODO: replace with v2.5 structs
+  validators: VDBSummaryValidators,
+  absolute: boolean,
+  groupId?: number,
+  timeFrame?: SummaryTimeFrame
+  context: DashboardValidatorContext,
+  dashboardKey?: DashboardKey,
+  isTooltip?: boolean
+}
+const props = defineProps<Props>()
+
+const { t: $t } = useI18n()
+const { groups } = useValidatorDashboardGroups()
+
+const dialog = useDialog()
+
+const openValidatorModal = () => {
+  dialog.open(DashboardValidatorSubsetModal, {
+    data: {
+      context: props.context,
+      timeFrame: props.timeFrame,
+      groupName: groupName.value,
+      groupId: props.groupId,
+      dashboardKey: props.dashboardKey
+    }
+  })
+}
+
+const groupName = computed(() => {
+  return getGroupLabel($t, props.groupId, groups.value)
+})
+
+const mapped = computed(() => {
+  const list: { count: number, key: string }[] = []
+  const addCount = (key: string, count?: number) => {
+    if (count) {
```

Comment:
If `count` is `0`, the corresponding Power icon will not be displayed. Oder?
As a user, I would like to always see the 3 icons, to see that I have 0 slashing or exited validator, so I know that the UI will tell me if it happens one day.

Reply:
That is by design. But I enhanced the logic for the absolute check (where we usually display only online validators) to display offline/exited if there is no online validator .